templates/apps/sumo-aws-apps/network-firewall/networkfw.template.yaml

```diff
@@ -698,6 +714,9 @@ Resources:
       SumoDeployment: !Ref SumoDeployment
 
 Outputs:
+  QSS3BucketName:
```

Comment:
Why did we add QSS3BucketName in output ? if not required then we can delete it.

Reply:
i use Output QSS3BucketName replace WaitCondition